templates/csharp/custom-copilot-weather-agent/appPackage/manifest.json.tpl

```diff
@@ -43,8 +53,8 @@
                             "description": "How can you help me?"
                         },
                         {
-                            "title": "Forecast the weather tomorrow.",
-                            "description": "Can you forecast the tomorrow weather for me?"
+                            "title": "Forecast the weather tomorrow in San Francisco.",
```

Comment:
The old prompt here can not work greatly, since the agent will ask for the location.